app/Http/Controllers/AsistenController.php

```diff
@@ -119,8 +119,8 @@ public function show($asisten_id)
         if(count($allAsistenData) !== 0){
 
             $ratingAsisten /= count($allAsistenData);
-            $gajiAsisten = $regHistoryCount * 25000;
-            $gajiAsisten += $intHistoryCount * 62500;
+            $gajiAsisten = $regHistoryCount * 31250;
```

Comment:
We should modify how we count `gaji` in the future XD

Reply:
Yeah this is correct, it should not be hardcoded all the time actually 😅